extract.go

```diff
@@ -293,6 +293,7 @@ func (se *selfExtractor) startup() {
 			se.exitCode <- 1
 		}
 	}
+	se.exitCode <- 0
```

Comment:
If the error is non-nil, `startup()` will have written a value in `exitCode` above and will block the goroutine here. Not a huge issue but it would be cleaner if that line was only called in case of no error

Reply:
I fixed it